Apps/phpanorama/panorama.json

```diff
@@ -5,7 +5,7 @@
     "publisher": "Splunk",
     "type": "firewall",
     "main_module": "panorama_connector.py",
-    "app_version": "2.0.0",
+    "app_version": "3.0.0",
```

Comment:
Customers report that only 2 actions (blockUrl and unblockUrl) are not working as expected. 
Since this fix allows us to work with different API versions, I think we should update the major version here.
What do you think?